unblob/__init__.py

```diff
@@ -0,0 +1,4 @@
+# get a traceback on crashes, as well as the responsible signal
+import faulthandler
+
+faulthandler.enable()
```

Comment:
I would rather put this into some init function as this could override the faulthandler setting of the main application when unblob is used as a library at import time. Maybe somehow make it configurable like logging?

Not support important, just wondering here.

Reply:
I mostly agree with a YAGNI feeling: I have some doubts if anyone ever needs to improve upon it.

This is not important from the fix perspective, but helped a lot to investigate the issue of zombie processes and stalled unblob, and has 0 impact on run-time otherwise. So left it here to avoid the hunt without traceback next time.

Maybe I am simply ignorant (`faulthandler` was new to me) and it can be/is used in other context, then I would postpone the refactoring work, when it is needed, alternatively I can drop this hunk.

Reply:
How about enabling it only if verbose level is `-vvv` ?

Reply:
I have dropped it, it was not important, the functionality is available by prefixing the `unblob` command with `PYTHONFAULTHANDLER=1` like this:

```sh
$ PYTHONFAULTHANDLER=1 unblob firmware.bin
```